Makefile

```diff
@@ -53,6 +53,29 @@ export ARCH
 
 export DBUILD_ARGS=--build-arg DBUILD_DATE=${DBUILD_DATE} --build-arg DBUILD_REPO_URL=${DBUILD_REPO_URL} --build-arg DBUILD_SITE_URL=${DBUILD_SITE_URL} --build-arg ARCH=${ARCH}
 
+# Specify the name for the binaries
+UPGRADE=upgrade
+
+# If there are any external tools need to be used, they can be added by defining a EXTERNAL_TOOLS variable 
+# Bootstrap the build by downloading additional tools
+.PHONY: bootstrap
+bootstrap:
+	@for tool in  $(EXTERNAL_TOOLS) ; do \
+		echo "Installing $$tool" ; \
+		go get -u $$tool; \
+	done
+
+.PHONY: clean
+clean: 
+	@echo '--> Cleaning directory...'
+	rm -rf bin
+	rm -rf ${GOPATH}/bin/${UPGRADE}
+	rm -rf ${GOPATH}/bin/${MIGRATE}
```

Comment:
`${MIGRATE}` is not defined.

Reply:
It is in another script included below `include ./build/migrate/Makefile.mk`. I will move this up.